OpenRA.Game/Map/ProjectedCellLayer.cs

```diff
@@ -27,6 +29,11 @@ public int Index(PPos uv)
 			return uv.V * Size.Width + uv.U;
 		}
 
+		public PPos PPosFromIndex(int index)
```

Comment:
IMO this is a little awkward. Can we instead define `Index` as a property on `PPos`, and likewise a static `FromIndex`?

Reply:
I agree that it is a little awkward